metaflow/sidecar.py

```diff
@@ -56,13 +56,14 @@ def start(self):
 
         if (self.__worker_type is not None and \
                 self.__worker_type.startswith(NULL_SIDECAR_PREFIX)) or \
-                platform.system() == 'Darwin':
-            self.__poller = NullPoller()
+                (platform.system() == 'Darwin' and sys.version_info < (3, 0)):
+                # if on darwin and running python 2 disable sidecars
+                # there is a bug with importing poll from select in some cases
+                self.__poller = NullPoller()
```

Comment:
Is it feasible to describe the bug - SO link or something here?

Reply:
don't have a good stackoverflow or github issues link but the error is:
```    
from select import poll
ImportError: cannot import name poll
```
I feel the comment is clear enough about where the issue is and since it only effects python2 on darwin i don't see a need to add anything here